v3/util/iam-proxy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+source /etc/environment
+
+if [ "${NODE_ROLE}" != "worker" ]; then
+    exit 0
+fi
+
+docker pull "behance/iam-docker:latest"
```

Comment:
we should really start adding the full-qualified registry in these "naked pulls" as we start down the ECR rabbit hole (or make it a parameter)